docs/dask.md

```diff
@@ -38,14 +38,36 @@ smallbaselineApp.py -t smallbaselineApp.cfg
 
 `numWorkers = all` will allocate `multiprocessing.cpu_count()` number of workers to the dask computation (for local cluster only). If the specified number of workers exceeds system resources, `multiprocessing.cpu_count()/2` number of workers will be submitted instead to avoid overtaxing local systems.
 
-#### 1.3 Runtime performance test on Stampede2 ####
+#### 1.3 Testing using example data ####
+
+Download and run the FernandinaSenDT128 example data; then run with and without local clasuter:
+
+```
+export OMP_NUM_THREADS=1
+cd FernandinaSenDT128/mintpy
+ifgram_inversion.py inputs/ifgramStack.h5 -w no --cluster no 
+ifgram_inversion.py inputs/ifgramStack.h5 -w no --cluster local --num-worker 8
+```
+
+A typical run time without local cluster is 30 secs and with 8 workers 11.4 secs.
```

Comment:
@falkamelung please specify the machine parameter you are using for the reference of "a typical run" please.

On my laptop (macOS 10.14 with 6 Intel i7 cores), w/o local cluster took 16 secs, w 8 workers took 30 secs, w 4 workers took 14 secs.